app/src/test/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntryLogicTest.kt

```diff
@@ -192,28 +193,6 @@ class CustomPrescriptionEntryLogicTest {
     verify(uiActions).setDosage(prescription.dosage)
   }
 
-  @Test
-  fun `when sheet is opened in edit mode and save is clicked after making changes, then the prescription should be updated`() {
-    //given
-    val prescribedDrug = TestData.prescription(uuid = prescriptionUuid, name = "Atnlol", dosage = "20mg")
-    val updatedPrescribedDrug = prescribedDrug.copy(name = "Atenolol", dosage = "5mg")
-
-    whenever(prescriptionRepository.prescription(prescriptionUuid)).thenReturn(Observable.just(prescribedDrug))
-    whenever(prescriptionRepository.updatePrescription(updatedPrescribedDrug)).thenReturn(Completable.complete())
-    whenever(prescriptionRepository.prescriptionImmediate(prescriptionUuid)).thenReturn(prescribedDrug)
-
-    //when
-    createSheetForUpdatingPrescription(prescriptionUuid)
-    uiEvents.onNext(CustomPrescriptionDrugNameTextChanged("Atenolol"))
-    uiEvents.onNext(CustomPrescriptionDrugDosageTextChanged("5mg"))
-    uiEvents.onNext(SaveCustomPrescriptionClicked)
-
-    //then
-    verify(prescriptionRepository).updatePrescription(updatedPrescribedDrug)
-    verify(prescriptionRepository, never()).savePrescription(any(), any(), any(), any())
-    verify(uiActions).finish()
```

Comment:
We'll need to add two more tests to complete the verification:

1. When the custom prescription entry is saved, we emit the effect to finish the sheet.
2. When the finish sheet effect is received, we invoke the UI action.

Reply:
Done